### PR TITLE
Initialize the server.py with index() and add update method.

### PR DESCRIPTION
**Pull request by p2hang**
Initialize the server.py with index()

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  """ Intro page of the inventory API
+
+  This method will only return some welcome words.
+
+  Returns:
+    response: welcome words in json format and status 200
+
+  Todo:
+    * Finish the implementations.
+    * Write the tests for this.
```
Comment (pinocchiochang) [inline server.py:17]: Seems no need for test by far.

File: server.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+from flask import Flask
```
Comment (pinocchiochang) [inline server.py:1]: No a comment, but what kind of installation for you to install flask? pip?

**Review by pinocchiochang — COMMENTED** [review]
I haven't read the details to create api with python yet, but is it require a __main__?

**Review by omersolmazer — APPROVED** [review]
Looking good!

**Review by BobZcao — COMMENTED** [review]
Good.

**Review by BobZcao — APPROVED** [review]
Approve

**Review by pinocchiochang — APPROVED** [review]
LGTM

File: server.py (hunk 2 above)
Comment (p2hang) [reply server.py:1]: I think we need to add the flask to the requirements

File: server.py (hunk 1 above)
Comment (p2hang) [reply server.py:17]: Yes!

**Pull request merged**